CLI.md

```diff
@@ -57,6 +57,7 @@ This document contains the help content for the `linera` command-line program.
 * [`linera storage initialize`↴](#linera-storage-initialize)
 * [`linera storage list_namespaces`↴](#linera-storage-list_namespaces)
 * [`linera storage list_blob_ids`↴](#linera-storage-list_blob_ids)
+* [`linera storage list_chain_ids`↴](#linera-storage-list_chain_ids)
 * [`linera storage list_root_keys`↴](#linera-storage-list_root_keys)
```

Comment:
do we need to list root keys?

Reply:
I think this can be useful for debugging, but if you think not, then so be it.